### PR TITLE
fisrt version of plotting code

### PR DESCRIPTION
**Pull request by otizonaizit**

### CODE REVIEW

File: plot.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+from matplotlib.ticker import MultipleLocator
+
+data = np.loadtxt('stats', delimiter=',')
+years = data[:,0]
+sums = data[:,1]/1000
+diffs = data[:,2]/1000
+
+plt.style.use('ggplot')
```
Comment (keszybz) [inline plot.py:11]: I think this should be split out to a function, so that we can do more plots in the future.

File: plot.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ax1.xaxis.set_label_position('top')
+#plt.setp(ax1.get_xticklabels(), visible=False)
+#ax2.plot(years, diffs, '*')
+
+
+ax2 = plt.subplot(gs[1], sharex = ax1)
+ax2.fill_between(years, diffs, color='darkkhaki')
+#ax2.grid(True)
+#ax2.xaxis.set_major_locator(MultipleLocator(1))
+plt.subplots_adjust(hspace=.01)
```
Comment (keszybz) [inline plot.py:31]: Please add an axis label here.

**Review by keszybz — CHANGES_REQUESTED** [review]
I think a plot is good!

File: plot.py (hunk 1 above)
Comment (otizonaizit) [reply plot.py:11]: OK, good idea. I will take care of that

File: plot.py [hunk 3]
```diff
@@ -8,26 +8,30 @@
 sums = data[:,1]/1000
 diffs = data[:,2]/1000
 
-plt.style.use('ggplot')
 
-fig = plt.figure()
-gs = gridspec.GridSpec(2, 1)
+def func():
```
Comment (keszybz) [inline plot.py]: Hmm, the function name should be something more expressive. Maybe `plot_all_the_important_income_statistics()`.

File: plot.py [hunk 4]
```diff
@@ -8,26 +8,30 @@
...
-#ax2.xaxis.set_major_locator(MultipleLocator(1))
-plt.subplots_adjust(hspace=.01)
-plt.show()
 
+    ax2 = plt.subplot(gs[1], sharex = ax1)
+    ax2.fill_between(years, diffs, color='darkkhaki')
+    #ax2.grid(True)
+    #ax2.xaxis.set_major_locator(MultipleLocator(1))
+    plt.subplots_adjust(hspace=.01)
+    plt.show()
```
Comment (keszybz) [inline plot.py]: I think that *this* part should be inside of the function, but the preamble above should *not*. This way we can reuse the code to do subplots in a bigger plot.

**Review by keszybz — CHANGES_REQUESTED** [review]

File: plot.py (hunk 2 above)
Comment (keszybz) [reply plot.py:31]: This part is not resolved yet.

**Pull request closed without merging**